core/db/migrate/20221215151408_add_selected_locale_to_spree_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSelectedLocaleToSpreeUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :spree_users, :selected_locale, :string
```

Comment:
Take a look at this file @wjwitek :
`core/db/migrate/20220201103922_add_first_name_and_last_name_to_spree_users.rb`

Technically you can use Spree with a different authentication schema, and it's been historically possible to configure table name for spree users (or even not have it at all). 

It would be great to use the same method as in the file above to:
1. Determine whether we have a table for spree users at all in the current spree instalation
2. Determine what's the correct table name to add the column to